qiita_db/support_files/patches/python_patches/51.py

```diff
@@ -90,7 +90,8 @@ def transform_date(value):
             # converting to datetime
             for col in columns:
                 st_df[col] = st_df[col].apply(transform_date)
-            st.update(st_df)
+            # SHOULD WE FIX?
```

Comment:
Since this is a schema change, I think it's all right to break with convention and remove/inactivate this.